mne/channels/montage.py

```diff
@@ -624,6 +633,44 @@ def read_dig_montage(hsp=None, hpi=None, elp=None, point_names=None,
         hsp = np.array(hsp) if len(hsp) else None
         elp = np.array(elp) if len(elp) else None
         coord_frame = 'head'
+    elif egi is not None:
+        if not all(x is None for x in (hsp, hpi, elp, point_names, fif)):
+            raise ValueError('hsp, hpi, elp, point_names, fif must all be '
+                             'None if egi is not None')
+        _check_fname(egi, overwrite=True, must_exist=True)
+        if not transform:
+            warn('Dig montage will not be in Neuromag head space. Set '
+                 'transform to True to place in correct coordinate space.')
```

Comment:
This warning isn't really specific to `egi`. We should probably emit it whenever `coord_frame == 'unknown'`, or not at all